packages/nuxi/bin/nuxi.mjs

```diff
@@ -1,3 +1,33 @@
 #!/usr/bin/env node
-process._startTime = Date.now()
-import('../dist/cli.mjs').then(r => (r.default || r).main())
+import { fileURLToPath } from 'node:url'
+import { execa } from 'execa'
```

Comment:
This file isn't bundled. Could we perhaps use `child_process`?

Reply:
IIRC, using `child_process` we will need to handle quite a lot of edge cases across platforms (path, windows, etc.)

Refactored to use the bundled execa. Thanks